spark/src/main/java/org/apache/accumulo/spark/CopyPlus5K.java

```diff
@@ -76,8 +78,20 @@ private static void cleanupAndCreateTables(Properties props) throws Exception {
         client.tableOperations().delete(outputTable);
       }
       // Create tables
-      client.tableOperations().create(inputTable);
-      client.tableOperations().create(outputTable);
+      try {
+        client.tableOperations().create(inputTable);
+      } catch (TableExistsException e) {
+        low.error("Something went wrong. Table '{}' should have been deleted prior to creation "
+            + "attempt!", inputTable);
+        return;
+      }
+      try {
+        client.tableOperations().create(outputTable);
+      } catch (TableExistsException e) {
+        low.error("Something went wrong. Table '{}' should have been deleted prior to creation "
```

Comment:
Should this be log (instead of low)

Reply:
Yes :) Thanks for the catch. It was not caught by compile since this class is not in the src path of the examples.